app/js/event/event.filter.js

```diff
@@ -11,13 +11,19 @@
 
     return function(x) {
       var distance;
-      if(isNaN(x)){
+      if (isNaN(x)) {
         return x;
-      } else {
+      }
+      else if (x > 1) {
         distance = x.toFixed(2);
-        return distance;
+        return distance + "km";
```

Comment:
This is not the preferred way to display the units. Could you check out how units are handled e.g. in the built-in [currency](https://docs.angularjs.org/api/ng/filter/currency) filter and do the same in this filter?